pages/common/ssh-agent.md

```diff
@@ -0,0 +1,20 @@
+# ssh-agent
+
+> Spawn an ssh-agent process
+> ssh-agent will hold your ssh key decrypted in memory until evicted
+> Add and manage keys with `ssh-add`
+
+- Start an SSH Agent for the current shell
+
+`eval $(ssh-agent)`
+
+- Kill the current agent
+
+`ssh-agent -k`
+
+- Manually connect to an existing agent (rare)
+
+`
+export SSH_AUTH_SOCK={{ssh-agent socket}}
+export SSH_AGENT_PID={{ssh-agent pid}}
```

Comment:
It should be one command per example.

Reply:
I think you need to have both of these to get the effect you want.

Reply:
Indeed you do, @nibalizer. I think that the first example is sufficient though - but we could tweak the page description I guess.